README.md

```diff
@@ -49,42 +49,42 @@ composer run phpcs -- --report=summary
 ```
 
 ## Extending the ruleset
+
 You can create a custom ruleset for your project that extends or customizes these rules by creating your own  `phpcs.xml` or `phpcs.xml.dist` file in your project, which references these rules, like this:
 
-```
+```xml
 <?xml version="1.0"?>
 <ruleset>
-	<description>Example project ruleset</description>
-
-    <!-- Include Alley Rules -->
-    <rule ref="Alley-Interactive" />
+  <description>Example project ruleset</description>
 
-    <!-- Project customizations go here -->
+  <!-- Include Alley Rules -->
+  <rule ref="Alley-Interactive" />
 
+  <!-- Project customizations go here -->
 </ruleset>
 ```
 
-# Change Log
+## Change Log
 
 This project adheres to [Keep a CHANGELOG](https://keepachangelog.com/en/1.0.0/).
 
-## 0.4.0-alpha
+### 0.4.0
```

Comment:
Should probably update this to note that PHPCompat is now configured for PHP 8.0, and note any dependency updates that get applied before we cut a new release.